app/nslc/serializers.py

```diff
@@ -33,9 +40,12 @@ class GroupSerializer(serializers.ModelSerializer):
     class Meta:
         model = Group
         fields = (
-            'name', 'id', 'description', 'channels',
+            'name', 'id', 'description',
             'created_at', 'updated_at', 'user', 'organization',
-            'auto_include_channels', 'auto_exclude_channels'
+            'share_all', 'share_org',
+            'channels_count', 'auto_include_channels_count',
+            'auto_exclude_channels_count',
+            'channels', 'auto_include_channels', 'auto_exclude_channels'
```

Comment:
If I'm understanding it correctly the `channels`, `auto_include_channels`, and `auto_exclude_channels` would only be for writing, but wouldn't be returned in a request. And then the opposite for the _count versions. Is that right?

Reply:
Yes - for the group serializer used for list & put/post you can't write to the counts & won't see the actual channels. The full detailed channels will be returned in the detail view only.